container/types.proto

```diff
@@ -35,12 +35,33 @@ message Container {
   // empty. Containers with duplicated attribute names or attributes with empty
   // values will be considered invalid.
   //
-  // There are some "well-known" attributes affecting system behaviour:
+  // There are some "well-known" attributes affecting system behaviour.
```

Comment:
Originally, here we describe two types of attributes: system and user defined. Maybe it would be better not to break this list into three lists and add more information about NFT in inline comments for each attribute?

Reply:
I don't see how NFT integration summary fits into _five_ different attributes. 

On top we described attributes in general, then "well-known" attributes, that include:
- sub network attribute (won't be used in early N3 though),
- NFT attributes,
- application specific attributes. 

This structure looks fine for me.

Reply:
OK, let's keep this structure.